third-party/tidb-operator/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@kubernetes-models/tidb-operator",
+  "description": "TiDB Operator models",
+  "version": "0.0.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tommy351/kubernetes-models-ts.git"
+  },
+  "homepage": "https://github.com/tommy351/kubernetes-models-ts/tree/master/third-party/tidb-operator",
+  "author": "Jay Chung <ken8203@gmail.com>",
+  "license": "MIT",
+  "main": "index.js",
+  "module": "index.mjs",
+  "types": "index.d.ts",
+  "sideEffects": false,
+  "scripts": {
+    "build": "crd-generate && publish-scripts build",
+    "prepack": "publish-scripts prepack"
+  },
+  "publishConfig": {
+    "access": "public",
+    "directory": "dist"
+  },
+  "keywords": [
+    "kubernetes",
+    "kubernetes-models",
+    "tidb-operator"
+  ],
+  "engines": {
+    "node": ">=14"
+  },
+  "dependencies": {
+    "@kubernetes-models/apimachinery": "workspace:^",
+    "@kubernetes-models/base": "workspace:^",
+    "@kubernetes-models/validate": "workspace:^",
+    "tslib": "^2.4.0"
+  },
+  "devDependencies": {
+    "@kubernetes-models/crd-generate": "workspace:^",
+    "@kubernetes-models/publish-scripts": "workspace:^",
+    "vitest": "^0.29.8"
+  },
+  "crd-generate": {
+    "input": [
```

Comment:
- According to [TiDB Operator documentation](https://docs.pingcap.com/tidb-in-kubernetes/stable/get-started#install-tidb-operator-crds), it should be fine to replace these files with this one instead.

  ```
  https://raw.githubusercontent.com/pingcap/tidb-operator/v1.4.4/manifests/crd.yaml
  ```
- I am unsure if we should use the beta version here since it might contain unstable changes. Are there any new features you want to use in the beta version?

Reply:
Wow, good to know that. And, no, I should just use v1.4.4.